pkg/codegen/testing/test/testdata/akamai/java/src/main/java/com/pulumi/akamai/ProviderArgs.java

```diff
@@ -18,14 +18,14 @@ public final class ProviderArgs extends com.pulumi.resources.ResourceArgs {
 
     public static final ProviderArgs Empty = new ProviderArgs();
 
-    @Import(name="cacheEnabled", json=true)
```

Comment:
Reminder here to remove `json` from the ImportAnnotation or at least mark deprecated.